tdh/resource_user.go

```diff
@@ -234,7 +279,7 @@ func (r *userResource) Create(ctx context.Context, req resource.CreateRequest, r
 	// Map response body to schema and populate Computed attribute values
 	createdUser := &(*users.Get())[0]
 
-	if saveFromUserResponse(&ctx, &resp.Diagnostics, &plan, createdUser) != 0 {
+	if saveFromUserResponse(&ctx, &resp.Diagnostics, &plan, createdUser, r.client.Root.IsSre) != 0 {
```

Comment:
convert it into a instance method: `r.saveFrom..`, you will have access to `r.Root.IsSRE`